cax/cax.json

```diff
@@ -3,76 +3,78 @@
     "name": "xenon1t-daq",
     "hostname": "xenon1t-daq.lngs.infn.it",
     "receive": null,
-    "directory": "/data/xenon/raw/",
+    "dir_raw": "/data/xenon/raw/",
+    "dir_processed": null,
     "upload_options": [],
-    "download_options": [],
-    "username": "transfer"
+    "download_options": [
+      "midway-login1"
```

Comment:
It's just temporary since I didn't have time to figure out a network thing.  I want to make sure xenon1t-daq has all the same data as eb0 so I can delete eb0.  I'll remove this download once I delete eb0.